mlflow/sklearn/__init__.py

```diff
@@ -185,7 +196,10 @@ def save_model(
         mlflow_model: :py:mod:`mlflow.models.Model` this flavor is being added to.
         serialization_format: The format in which to serialize the model. This should be one of
             the formats listed in
-            ``mlflow.sklearn.SUPPORTED_SERIALIZATION_FORMATS``. The Cloudpickle
+            ``mlflow.sklearn.SUPPORTED_SERIALIZATION_FORMATS``.
+            The default format is ``mlflow.sklearn.SERIALIZATION_FORMAT_SKOPS``
+            which guarantees security for model loading.
+            The Cloudpickle
```

Comment:
We can list all the available options. Instead of referencing `mlflow.sklearn.SUPPORTED_SERIALIZATION_FORMATS` or `mlflow.sklearn.SERIALIZATION_FORMAT_SKOPS`.